hummingbird/ml/_container.py

```diff
@@ -69,6 +69,17 @@ def forward(self, *inputs):
             else:
                 return list(variable_map[output_name] for output_name in self.output_names)
 
+
+class PyTorchBackendModelTransformer(PyTorchBackendModel):
+    def transform(self, *inputs):
+        """
+        Utility functions used to emulate the behavior of the Sklearn API.
+        On data transforerms it returns trandormed output data
+        """
+        return self.forward(*inputs).cpu().numpy().flatten()
```

Comment:
I know this is correct and necessary but....oh numpy.

Reply:
Yea.